src/bun.js/api/JSTranspiler.zig

```diff
@@ -723,6 +723,18 @@ fn transformOptionsFromJSC(globalObject: JSC.C.JSContextRef, temp_allocator: std
         transpiler.runtime.replace_exports = replacements;
     }
 
+    if (object.get(globalThis, "logLevel")) |logLevel| {
+        const logLevelString = logLevel.toBunString(globalThis);
+        defer logLevelString.deref();
+        const key = logLevelString.toOwnedSlice(allocator) catch bun.outOfMemory();
+        if (logger.Log.Level.Map.get(key)) |level| {
```

Comment:
We don't need to clone the string here. I'm going to merge this and then fix that